curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f5486b8fd4b71633f69b0.md

```diff
@@ -29,16 +29,16 @@ Your `p` element should come after your `.small-text` element.
 assert(document.querySelector('.small-text')?.nextElementSibling?.localName === 'p');
 ```
 
-Your `p` element should have the text `Calories`.
+Your `span` element should come after your `.left-container` element.
```

Comment:
Checking the location of the span and p before checking their contents means we can check their contents however we like, making those tests less fragile.